OrbitGl/GlCanvas.h

```diff
@@ -5,6 +5,7 @@
 #ifndef ORBIT_GL_GL_CANVAS_H_
 #define ORBIT_GL_GL_CANVAS_H_
 
+#include "AccessibleInterface.h"
```

Comment:
And then this will be included with `#include "OrbitAccessibility/AccessibleInterface.h"`.